build.gradle

```diff
@@ -146,6 +153,24 @@ def convertGitBranch = { gitBranch ->
     }
 }
 
+createExe {
+    dontWrapJar = true
+    stayAlive = true
+    version = "${versionMajor}.${versionMinor}"
+    textVersion = displayVersion
+    companyName = orgName
+    jreMinVersion = "${targetCompatibility}.0" // launch4j requires it's in the form x.x.x, thus this currently renders to 1.8.0
```

Comment:
I'm sorry for `build.sbt`, got confused with our work build system here - I totally meant the `build.gradle`. In addition, you are right with `1.8.0_40` (as you can see [here](https://github.com/MovingBlocks/TerasologyLauncher/blob/develop/build.gradle#L53)). 

Please excuse this little inconvenience from my side ☺️